sdk/identity/identity/CHANGELOG.md

```diff
@@ -14,14 +14,15 @@ This release continues with the changes from `1.2.4` and `1.2.4-beta.1`.
 - Migrated the `InteractiveBrowserCredential`, `DeviceCodeCredential`, `ClientSecretCredential`, `ClientCertificateCredential` and `UsernamePasswordCredential` to the latest MSAL.
   - This update improves caching of tokens, significantly reducing the number of network requests.
 - Credentials `InteractiveBrowserCredential`, `DeviceCodeCredential` and `UsernamePasswordCredential` now can:
+  - Receive a `tokenCachePersistenceOptions` parameter to specify persitence caching of the credentials used to authenticate. This feature will not have support for Node 8 as part of this beta. This feature uses PNPM on Windows, it tries to use the Keychain on OSX and the Keyring on Linux, and if the user sets `allowUnencryptedStorage` to true in the `tokenCachePersistenceOptions`, it allows to fall back to an unprotected file if neither the Keychain nor the Keyring are available.
```

Comment:
I don't think this has anything to do with PNPM on Windows.  PNPM is just part of our internal build system, not relevant to consumers of this package.

Reply:
No no, the persistence feature does use PNPM on Windows. I could avoid mentioning it though! I also mention this on the comments for this optional parameter on the credentials. @schaabs

Reply:
Oh my bad I meant DPAPI. I'll fix this.